src/pipelines/rust/mod.rs

```diff
@@ -456,13 +456,16 @@ impl RustApp {
         // checking for errors, otherwise the dir will never be ignored. If we attempt to do
         // this pre-build, the canonicalization will fail and will not be ignored.
         if let Some(chan) = &mut self.ignore_chan {
-            let _ = chan.try_send(
-                self.manifest
-                    .metadata
-                    .target_directory
-                    .clone()
-                    .into_std_path_buf(),
-            );
+            let target_dir = self
+                .manifest
+                .metadata
+                .target_directory
+                .clone()
+                .into_std_path_buf()
+                .canonicalize()
+                .expect("valid path");
```

Comment:
Sorry I didn't spot this before: using `expect` will just terminate the application. Leaving one puzzled what happened. In the spirit of ignoring errors in the next line, why not just ignore (and log) this one too?